editor/src/clj/editor/fs.clj

```diff
@@ -103,6 +103,11 @@
       (catch Exception _
         false))))
 
+(defn below-directory?
+  "Returns true if the file system entry is located below the directory."
+  [^File entry ^File directory]
+  (-> entry .toPath .toAbsolutePath (.startsWith (-> directory .toPath .toAbsolutePath))))
```

Comment:
I wonder how well this handles case-insensitive file systems. The `editor.resource/relative-path` fn has some special handling in this area.